### DISCUSSION TITLE
Good First Issue new Contributors

### DISCUSSION THREAD

**Discussion by salahlalami** [idurar/idurar-erp-crm#287] [Announcements]
Here new Good First Issue new Contributors : [ https://github.com/idurar/idurar-erp-crm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22]( https://github.com/idurar/idurar-erp-crm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)

Attention Developers! Join us in contributing to the IDURAR open-source project based on Node.js and React.js. We have a collection of "Good First Issue" tickets specially curated for new contributors like you.

By tackling these issues, you'll get hands-on experience working with popular web development technologies and gain valuable insights into building robust applications. Whether you're a beginner or an experienced developer looking to expand your skills, this is a perfect opportunity to jump into open source development.

These "Good First Issue" tickets are designed to be beginner-friendly, with detailed instructions and guidance provided by our friendly community. You can choose from a variety of tasks, such as adding new features, fixing bugs, improving user interfaces, or enhancing performance.

Joining our project means becoming part of a supportive community where you can learn, collaborate, and showcase your skills to potential employers or clients. It's an excellent way to build a strong portfolio and gain recognition within the developer community.

So why wait? Visit our GitHub repository and start exploring the "Good First Issue" tickets. We can't wait to see your contributions and help you grow as a developer. Let's create something amazing together!

Remember, every contribution matters, no matter how big or small. Come and join us at IDURAR and make a difference in the open-source world. Happy coding!

Feel free to ask me if you have any questions or need assistance getting started.

Here new Good First Issue new Contributors : [ https://github.com/idurar/idurar-erp-crm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22]( https://github.com/idurar/idurar-erp-crm/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)

**Comment by arpitnakrani** [comment]
Not able to run FE its giving this tried - downgrade npm and node version , here i  am attaching ss of error ,  thank you in advance for your response @salahlalami 
![image](https://github.com/idurar/idurar-erp-crm/assets/97463573/935241a6-92a4-477b-9343-31c596711f70)
